src/index.ts

```diff
@@ -7,29 +7,19 @@ function VitePluginMarkdown(userOptions: Options = {}): Plugin {
   const options = resolveOptions(userOptions)
   const markdownToVue = createMarkdown(options)
 
-  let vuePlugin: Plugin
-
   return {
     name: 'vite-plugin-md',
     enforce: 'pre',
-    configResolved(config) {
-      vuePlugin = config.plugins.find(p => p.name === 'vite:vue')!
-      if (!vuePlugin)
-        throw new Error('[vite-plugin-md] no vue plugin found, do you forget to install it?')
-    },
     transform(raw, id) {
       if (id.endsWith('.md'))
```

Comment:
In think this extension should also be configurable, but i'll make another pull request when this one is merged.

Reply:
something like `include`/`exclude` regexes like in vue plugin.

```typescript
include?: string | RegExp | (string | RegExp)[]
exclude?: string | RegExp | (string | RegExp)[]
```